Kernel/Arch/x86/common/Processor.cpp

```diff
@@ -663,7 +663,20 @@ UNMAP_AFTER_INIT void Processor::initialize(u32 cpu)
     if (cpu == 0) {
         VERIFY((FlatPtr(&s_clean_fpu_state) & 0xF) == 0);
         asm volatile("fninit");
-        if (has_feature(CPUFeature::FXSR))
+        // Initialize AVX state
+        if (has_feature(CPUFeature::XSAVE) && has_feature(CPUFeature::AVX)) {
+            asm volatile("mov $7, %%eax"
+                         :
+                         :
+                         : "eax");
+            asm volatile("xor %%edx, %%edx"
+                         :
+                         :
+                         : "edx");
+            asm volatile("xsave %0"
+                         : "=m"(s_clean_fpu_state));
+        }
```

Comment:
```suggestion
            asm volatile("xsave %0"
                         : "=m"(s_clean_fpu_state)
                         : "a"(7), "d"(0));
```
Similarly below

Reply:
Whoa, that makes life _much_ easier my God